packages/mui-base/src/index.d.ts

```diff
@@ -14,12 +14,6 @@ export * from './ClickAwayListener';
 export { default as unstable_composeClasses } from './composeClasses';
 export * from './composeClasses';
 
-export { default as generateUtilityClass } from './generateUtilityClass';
```

Comment:
As far as I know, the X components depend on this API. Could you double check please before removing the API?

Reply:
Or they depend on the @mui/utils, not sure to be honest.

Reply:
The PR in X is now merged in, so I believe this one is safe to be merged in as well.